README.md

```diff
@@ -42,15 +42,17 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Chrispaix KAZE**
 
-👤 **Author2**
+- GitHub: [@ChrispaixK](https://github.com/ChrispaixK)
+- Twitter: [@ChrispaixK](https://twitter.com/ChrispaixK)
+- LinkedIn: [Chrispaix KAZE](https://www.linkedin.com/in/chrispaix-kaze-70445a175/)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Author1**
+
+- GitHub: (https://github.com/rotshidzw)
+- Twitter: (https://twitter.com/footballlz )
+- LinkedIn: (https://www.linkedin.com/in/rochidzwa-chester-8062b6211/)
 
 ## 🤝 Contributing
 
```

Comment:
- [ ] It will be nice to update your `Readme file` with the information for this milestone. If information is not available for particular sub heading in this template, it will nice to remove sub heading as this will make your `Readme file` look clean, informative and more professional. Kindly update each sub heading with information pertaining to this milestone. Remember to update your project title, and add steps to clone and run your project locally.